### PR TITLE
Rebrand refinement

### PR DESCRIPTION
**Pull request by davidrapson**
Work in progress pull request to start rationalising typography, colours, and components for the rebrand.

### CODE REVIEW

File: assets/sass/_functions.scss [hunk 1]
```diff
@@ -2,6 +2,23 @@
 // Functions
 // ========================================================================= //
 
+// =========================================================================
+// Functions: Map deep get
+// ========================================================================= //
+
+/// Map deep get
```
Comment (mattandrews) [inline assets/sass/_functions.scss]: [Ummmmmm....](https://github.com/biglotteryfund/blf-alpha/blob/master/assets/sass/_functions.scss#L66)

File: assets/sass/_functions.scss [hunk 2]
```diff
@@ -18,12 +35,12 @@
...
 // ========================================================================= //
 
-@function font-stack($key) {
-    @if type-of(map-get($font-stacks, $key)) == string {
-        @return unquote(map-get($font-stacks, $key));
-    } @else {
-        @error "font-stack #{$key} is not defined";
-    }
+@function font-stack($category, $variant: 'default') {
+    @return unquote(map-deep-get($typograhy, $category, 'font-stacks', $variant));
```
Comment (mattandrews) [inline assets/sass/_functions.scss]: Typo on `typograhy` (below too).

File: assets/sass/_vars-rebrand.scss [hunk 3]
```diff
@@ -1,42 +1,32 @@
...
+   Global Variables: Typography
    ========================================================================= */
 
-// If updating font-stacks make sure to update in
-// `assets/js/head.js` and `gulpfile.js`
-$font-stacks: (
-    fallback: 'sans-serif',
-    display: 'caecilia, sans-serif',
-    body: 'caecilia-sans-text, Trebuchet MS, sans-serif'
+$typograhy: (
```
Comment (mattandrews) [inline assets/sass/_vars-rebrand.scss]: image

Comment (mattandrews) [reply assets/sass/_vars-rebrand.scss]: :trollface:

File: assets/sass/_functions.scss (hunk 1 above)
Comment (davidrapson) [reply assets/sass/_functions.scss]: ?

Comment (mattandrews) [reply assets/sass/_functions.scss]: Click the link! That function already exists in this file.

Comment (davidrapson) [reply assets/sass/_functions.scss]: lol

File: assets/sass/utilities/_utilities-text.scss [hunk 4]
```diff
@@ -14,14 +14,6 @@
     font-weight: 100 !important;
 }
 
-.u-weight-normal {
```
Comment (mattandrews) [inline assets/sass/utilities/_utilities-text.scss]: I just used this in my updates branch, can we keep it around for now?

File: controllers/routes.js [hunk 5]
```diff
@@ -51,10 +51,6 @@ const toplevel = {
                 template: 'static-pages/region'
             })
         },
-        {
-            path: '/contact',
```
Comment (mattandrews) [inline controllers/routes.js]: Does this just come from the CMS now?

**Comment by mattandrews** [comment]
Looks like some colour changes (new vs old):


### Homepage
image
image
image

**Comment by mattandrews** [comment]
image

Lost the dividers in the nav for pages without heroes??

**Comment by mattandrews** [comment]
image

This looks a bit ugly IMO (was quite nice matching the button colours to the actual files you're downloading)

File: controllers/routes.js (hunk 5 above)
Comment (davidrapson) [reply controllers/routes.js]: This got moved to be a "top-level" section, see later on in this routes file.

**Comment by davidrapson** [comment]
> This looks a bit ugly IMO (was quite nice matching the button colours to the actual files you're downloading)

I don't disagree, but I'm OK with this given that this whole thing will change pretty soon with the new logo. That said, it might be possible to reinstate this anyway as we are still keeping the blue accent colour around.

**Comment by davidrapson** [comment]
Apologies I think I missed pushing a couple of commits when asking you to review so the button colour thing was a red herring but the rest are great spots. Fixed the regression with the header border and also reinstated the logo button colours.

**Pull request merged**